--- "a"
+++ "b"
@@ -63,21 +63,6 @@
     assert(offset + count <= data.size);
     memcpy((void *)(data.ptr + offset), (void *)src, count);
     
-    /*
-    if (writeThrough && file) {
-        
-        printf("Write through...\n");
-        
-        // Move to the correct position
-        file.seekp(offset, std::ios::beg);
-
-        // Write the data to the stream
-        file.write((char *)(data.ptr + offset), count);
-        
-        // Update the file on disk
-        file.flush();
-    }
-    */
 }
 
 ByteView
