--- "a"
+++ "b"
@@ -12,7 +12,6 @@
 #include "Agnus.h"
 #include "MsgQueue.h"
 #include "Paula.h"
-#include "RemoteManager.h"
 #include "SerialPort.h"
 #include "Amiga.h"
 #include "MidiManager.h"
@@ -169,8 +168,6 @@
     // Send the byte to the null modem cable or MIDI
     if (serialPort.config.device == SerialPortDevice::MIDI) {
         amiga.midiManager.sendByte(transmitBuffer & 0xFF);
-    } else {
-        remoteManager.serServer << char(transmitBuffer);
     }
 
     // Move the contents of the transmit buffer into the shift register
