--- "a"
+++ "b"
@@ -298,8 +298,6 @@
         // Inform the GUI if the record buffer had been empty
         if (incoming.length() == 1) msgQueue.put(Msg::SER_IN);
 
-        // Inform RetroShell
-        if (config.verbose) dumpByte(byte);
     }
 }
 
@@ -316,38 +314,12 @@
         // Inform the GUI if the record buffer had been empty
         if (outgoing.length() == 1) msgQueue.put(Msg::SER_OUT);
 
-        // Inform RetroShell
-        if (config.device == SerialPortDevice::RETROSHELL || config.device == SerialPortDevice::COMMANDER) dumpByte(byte);
     }
 }
 
 void
 SerialPort::dumpByte(int byte)
 {
-    char c = char(byte);
-
-    if (config.device == SerialPortDevice::RETROSHELL) {
-
-        if (isprint(c) || c == '\n') {
-            retroShell << c;
-        }
-    }
-
-    if (config.device == SerialPortDevice::COMMANDER) {
-
-        switch (c) {
-
-            case '\n':
-
-                retroShell.press(RSKey::RETURN);
-                break;
-
-            default:
-
-                if (isprint(c)) retroShell.press(c);
-                break;
-        }
-    }
 }
 
 }
