--- "a"
+++ "b"
@@ -100,9 +100,6 @@
     // auto  diameter = eadf.getDiameter();
     // auto  density  = eadf.getDensity();
 
-    if constexpr (debug::MFM_DEBUG)
-        fprintf(stderr, "Encoding Amiga disk with %ld tracks\n", tracks);
-
     // Create an empty ADF
     // auto adf = make_unique<ADFFile>(ADFFile::fileSize(diameter, density, tracks));
 
@@ -164,8 +161,6 @@
 {
     if (eadf.typeOfTrack(t) == 0) {
 
-        if constexpr (debug::MFM_DEBUG)
-            fprintf(stderr, "Encoding standard track %ld\n", t);
 
         auto numBits = eadf.usedBitsForTrack(t);
         assert(numBits % 8 == 0);
@@ -180,8 +175,6 @@
 {
     if (eadf.typeOfTrack(t) == 1) {
 
-        if constexpr (debug::MFM_DEBUG)
-            fprintf(stderr, "Encoding extended track %ld\n", t);
 
         auto numBits = eadf.usedBitsForTrack(t);
         assert(numBits % 8 == 0);
@@ -252,8 +245,6 @@
         }
     }
 
-    if constexpr (debug::MFM_DEBUG)
-        fprintf(stderr, "Wrote %td bytes\n", p - eadf.data.ptr);
 }
 
 void
