--- "a"
+++ "b"
@@ -74,12 +74,12 @@
         &ciaA,
         &ciaB,
         &mem,
-        &cpu,
-        &logicAnalyzer,
-        &remoteManager,
-        &retroShell,
-        &osDebugger,
-        &regressionTester
+        &cpu//,
+        // &logicAnalyzer,
+        // &remoteManager,
+        // &retroShell,
+        // &osDebugger,
+        // &regressionTester
     };
 
     info.bind([this] { return cacheInfo(); } );
@@ -321,8 +321,8 @@
     }
 
     // Execute the setup script
-    retroShell.asyncExec("commander");
-    retroShell.asyncExecScript(ss);
+    // retroShell.asyncExec("commander");
+    // retroShell.asyncExecScript(ss);
 }
 
 void
@@ -826,7 +826,7 @@
 {
     loginfo(RUN_DEBUG, "_pause\n");
 
-    remoteManager.gdbServer.breakpointReached();
+    // remoteManager.gdbServer.breakpointReached();
     msgQueue.put(Msg::PAUSE);
 }
 
@@ -965,7 +965,7 @@
                 
             case Cmd::RSH_EXECUTE:
 
-                retroShell.exec();
+                // retroShell.exec();
                 break;
 
             case Cmd::FOCUS:
@@ -982,10 +982,10 @@
     if (cmdConfig) { msgQueue.put(Msg::CONFIG, isize(cmd.type)); }
 
     // Inform the GUI about new RetroShell content
-    if (retroShell.isDirty) { retroShell.isDirty = false; msgQueue.put(Msg::RSH_UPDATE); }
+    // if (retroShell.isDirty) { retroShell.isDirty = false; msgQueue.put(Msg::RSH_UPDATE); }
 
     // Update subcomponents
-    remoteManager.update();
+    // remoteManager.update();
 }
 
 void
