--- "a"
+++ "b"
@@ -112,10 +112,10 @@
     registerDefault(Opt::DMA_DEBUG_COLOR6,           0xFFFFFF00);
     registerDefault(Opt::DMA_DEBUG_COLOR7,           0xFF000000);
 
-    registerDefault(Opt::LA_PROBE0,                  (i64)Probe::NONE);
-    registerDefault(Opt::LA_PROBE1,                  (i64)Probe::NONE);
-    registerDefault(Opt::LA_PROBE2,                  (i64)Probe::NONE);
-    registerDefault(Opt::LA_PROBE3,                  (i64)Probe::NONE);
+    // registerDefault(Opt::LA_PROBE0,                  (i64)Probe::NONE);
+    // registerDefault(Opt::LA_PROBE1,                  (i64)Probe::NONE);
+    // registerDefault(Opt::LA_PROBE2,                  (i64)Probe::NONE);
+    // registerDefault(Opt::LA_PROBE3,                  (i64)Probe::NONE);
     registerDefault(Opt::LA_ADDR0,                   0);
     registerDefault(Opt::LA_ADDR1,                   0);
     registerDefault(Opt::LA_ADDR2,                   0);
@@ -213,30 +213,30 @@
 
     registerDefault(Opt::DIAG_BOARD,                 false);
 
-    registerDefaults(Opt::SRV_ENABLE,                 false,                  { (i64)ServerType::RSH });
-    registerDefaults(Opt::SRV_PORT,                   8081,                   { (i64)ServerType::RSH });
-    registerDefaults(Opt::SRV_PROTOCOL,               (i64)ServerProtocol::DEFAULT, { (i64)ServerType::RSH });
-    registerDefaults(Opt::SRV_VERBOSE,                true,                   { (i64)ServerType::RSH });
-
-    registerDefaults(Opt::SRV_ENABLE,                 false,                  { (i64)ServerType::RPC });
-    registerDefaults(Opt::SRV_PORT,                   8082,                   { (i64)ServerType::RPC });
-    registerDefaults(Opt::SRV_PROTOCOL,               (i64)ServerProtocol::DEFAULT, { (i64)ServerType::RPC });
-    registerDefaults(Opt::SRV_VERBOSE,                true,                   { (i64)ServerType::RPC });
-
-    registerDefaults(Opt::SRV_ENABLE,                 false,                  { (i64)ServerType::GDB });
-    registerDefaults(Opt::SRV_PORT,                   8083,                   { (i64)ServerType::GDB });
-    registerDefaults(Opt::SRV_PROTOCOL,               (i64)ServerProtocol::DEFAULT, { (i64)ServerType::GDB });
-    registerDefaults(Opt::SRV_VERBOSE,                true,                   { (i64)ServerType::GDB });
-
-    registerDefaults(Opt::SRV_ENABLE,                 false,                  { (i64)ServerType::PROM });
-    registerDefaults(Opt::SRV_PORT,                   8084,                   { (i64)ServerType::PROM });
-    registerDefaults(Opt::SRV_PROTOCOL,               (i64)ServerProtocol::DEFAULT, { (i64)ServerType::PROM });
-    registerDefaults(Opt::SRV_VERBOSE,                true,                   { (i64)ServerType::PROM });
-
-    registerDefaults(Opt::SRV_ENABLE,                 false,                  { (i64)ServerType::SER });
-    registerDefaults(Opt::SRV_PORT,                   8085,                   { (i64)ServerType::SER });
-    registerDefaults(Opt::SRV_PROTOCOL,               (i64)ServerProtocol::DEFAULT, { (i64)ServerType::SER });
-    registerDefaults(Opt::SRV_VERBOSE,                true,                   { (i64)ServerType::SER });
+    // registerDefaults(Opt::SRV_ENABLE,                 false,                  { (i64)ServerType::RSH });
+    // registerDefaults(Opt::SRV_PORT,                   8081,                   { (i64)ServerType::RSH });
+    // registerDefaults(Opt::SRV_PROTOCOL,               (i64)ServerProtocol::DEFAULT, { (i64)ServerType::RSH });
+    // registerDefaults(Opt::SRV_VERBOSE,                true,                   { (i64)ServerType::RSH });
+    // 
+    // registerDefaults(Opt::SRV_ENABLE,                 false,                  { (i64)ServerType::RPC });
+    // registerDefaults(Opt::SRV_PORT,                   8082,                   { (i64)ServerType::RPC });
+    // registerDefaults(Opt::SRV_PROTOCOL,               (i64)ServerProtocol::DEFAULT, { (i64)ServerType::RPC });
+    // registerDefaults(Opt::SRV_VERBOSE,                true,                   { (i64)ServerType::RPC });
+    // 
+    // registerDefaults(Opt::SRV_ENABLE,                 false,                  { (i64)ServerType::GDB });
+    // registerDefaults(Opt::SRV_PORT,                   8083,                   { (i64)ServerType::GDB });
+    // registerDefaults(Opt::SRV_PROTOCOL,               (i64)ServerProtocol::DEFAULT, { (i64)ServerType::GDB });
+    // registerDefaults(Opt::SRV_VERBOSE,                true,                   { (i64)ServerType::GDB });
+    // 
+    // registerDefaults(Opt::SRV_ENABLE,                 false,                  { (i64)ServerType::PROM });
+    // registerDefaults(Opt::SRV_PORT,                   8084,                   { (i64)ServerType::PROM });
+    // registerDefaults(Opt::SRV_PROTOCOL,               (i64)ServerProtocol::DEFAULT, { (i64)ServerType::PROM });
+    // registerDefaults(Opt::SRV_VERBOSE,                true,                   { (i64)ServerType::PROM });
+    // 
+    // registerDefaults(Opt::SRV_ENABLE,                 false,                  { (i64)ServerType::SER });
+    // registerDefaults(Opt::SRV_PORT,                   8085,                   { (i64)ServerType::SER });
+    // registerDefaults(Opt::SRV_PROTOCOL,               (i64)ServerProtocol::DEFAULT, { (i64)ServerType::SER });
+    // registerDefaults(Opt::SRV_VERBOSE,                true,                   { (i64)ServerType::SER });
 
     defaults.values["ROM_PATH"] = "";
     defaults.values["EXT_PATH"] = "";
