--- "a"
+++ "b"
@@ -16,7 +16,11 @@
 #include "Media.h"
 #include "MsgQueue.h"
 #include "CmdQueue.h"
-#include "OSDescriptors.h"
+#include "DeviceError.h"
+#include "Volume.h"
+#include "FileSystems/Amiga/FileSystem.h"
+#include "FileSystems/FSError.h"
+// #include "OSDescriptors.h"
 #include "utl/io.h"
 
 namespace vamiga {
@@ -1172,20 +1176,20 @@
     file.extractData(buffer);
 
     // Parse hunks
-    auto descr = ProgramUnitDescriptor(buffer);
+    // auto descr = ProgramUnitDescriptor(buffer);
     
     // Seek the code section and read the first instruction word
-    auto offset = descr.seek(HUNK_CODE);
-    if (!offset) throw CoreError(CoreError::HUNK_CORRUPTED);
-    u16 instr = HI_LO(buffer[*offset + 8], buffer[*offset + 9]);
+    // auto offset = descr.seek(HUNK_CODE);
+    // if (!offset) throw CoreError(CoreError::HUNK_CORRUPTED);
+    // u16 instr = HI_LO(buffer[*offset + 8], buffer[*offset + 9]);
     
     // Replace the first instruction word by a software trap
-    auto trap = cpu.debugger.swTraps.create(instr);
-    buffer[*offset + 8] = HI_BYTE(trap);
-    buffer[*offset + 9] = LO_BYTE(trap);
+    // auto trap = cpu.debugger.swTraps.create(instr);
+    // buffer[*offset + 8] = HI_BYTE(trap);
+    // buffer[*offset + 9] = LO_BYTE(trap);
     
     // Write the modification back to the file system
-    file.overwriteData(buffer);
+    // file.overwriteData(buffer);
 
     // Replace the old disk
     fs.flush();
