--- "a"
+++ "b"
@@ -104,10 +104,10 @@
     DMA_DEBUG_COLOR7,
     
     // Logic analyzer
-    LA_PROBE0,              ///< Probe on channel 0
-    LA_PROBE1,              ///< Probe on channel 1
-    LA_PROBE2,              ///< Probe on channel 2
-    LA_PROBE3,              ///< Probe on channel 3
+    // LA_PROBE0,              ///< Probe on channel 0
+    // LA_PROBE1,              ///< Probe on channel 1
+    // LA_PROBE2,              ///< Probe on channel 2
+    // LA_PROBE3,              ///< Probe on channel 3
     LA_ADDR0,               ///< Address for channel 0 (memory probing)
     LA_ADDR1,               ///< Address for channel 1 (memory probing)
     LA_ADDR2,               ///< Address for channel 2 (memory probing)
@@ -307,10 +307,10 @@
             case Opt::DMA_DEBUG_COLOR6:          return "DMA.DEBUG_COLOR6";
             case Opt::DMA_DEBUG_COLOR7:          return "DMA.DEBUG_COLOR7";
                 
-            case Opt::LA_PROBE0:                 return "LA.PROBE0";
-            case Opt::LA_PROBE1:                 return "LA.PROBE1";
-            case Opt::LA_PROBE2:                 return "LA.PROBE2";
-            case Opt::LA_PROBE3:                 return "LA.PROBE3";
+            // case Opt::LA_PROBE0:                 return "LA.PROBE0";
+            // case Opt::LA_PROBE1:                 return "LA.PROBE1";
+            // case Opt::LA_PROBE2:                 return "LA.PROBE2";
+            // case Opt::LA_PROBE3:                 return "LA.PROBE3";
             case Opt::LA_ADDR0:                  return "LA.ADDR0";
             case Opt::LA_ADDR1:                  return "LA.ADDR1";
             case Opt::LA_ADDR2:                  return "LA.ADDR2";
@@ -489,10 +489,10 @@
             case Opt::DMA_DEBUG_COLOR6:          return "CPU color";
             case Opt::DMA_DEBUG_COLOR7:          return "Memory refresh color";
                 
-            case Opt::LA_PROBE0:                 return "Probe on channel 0";
-            case Opt::LA_PROBE1:                 return "Probe on channel 1";
-            case Opt::LA_PROBE2:                 return "Probe on channel 2";
-            case Opt::LA_PROBE3:                 return "Probe on channel 3";
+            // case Opt::LA_PROBE0:                 return "Probe on channel 0";
+            // case Opt::LA_PROBE1:                 return "Probe on channel 1";
+            // case Opt::LA_PROBE2:                 return "Probe on channel 2";
+            // case Opt::LA_PROBE3:                 return "Probe on channel 3";
             case Opt::LA_ADDR0:                  return "Channel 0 memory address";
             case Opt::LA_ADDR1:                  return "Channel 1 memory address";
             case Opt::LA_ADDR2:                  return "Channel 2 memory address";
