--- "a"
+++ "b"
@@ -12,7 +12,7 @@
 #include "Agnus.h"
 #include "MsgQueue.h"
 #include "Paula.h"
-#include "RemoteManager.h"
+// #include "RemoteManager.h"
 #include "SerialPort.h"
 #include "Amiga.h"
 #include "MidiManager.h"
@@ -170,7 +170,7 @@
     if (serialPort.config.device == SerialPortDevice::MIDI) {
         amiga.midiManager.sendByte(transmitBuffer & 0xFF);
     } else {
-        remoteManager.serServer << char(transmitBuffer);
+        // remoteManager.serServer << char(transmitBuffer);
     }
 
     // Move the contents of the transmit buffer into the shift register
